selfdrive/controls/lib/longitudinal_mpc_lib/long_mpc.py

```diff
@@ -348,15 +348,9 @@ def update_with_xva(self, x, v, a):
     for i in range(N):
       self.solver.cost_set(i, "yref", self.yref[i])
     self.solver.cost_set(N, "yref", self.yref[N][:COST_E_DIM])
-    self.accel_limit_arr[:,0] = -10.
-    self.accel_limit_arr[:,1] = 10.
-    x_obstacle = 1e5*np.ones(N+1)
-    self.params = np.concatenate([self.accel_limit_arr,
-                                  x_obstacle[:, None],
-                                  self.prev_a[:,None]], axis=1)
+    self.params[:,3] = self.prev_a
```

Comment:
```suggestion
    self.params[:,3] = np.copy(self.prev_a)
```

Just to be sure I'll match `update` here.